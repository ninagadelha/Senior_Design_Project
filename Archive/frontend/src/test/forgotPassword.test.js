import renderer from "react-test-renderer";
import ForgotPassword from "../routes/forgotPassword";

jest.mock("../styles/about.css");
jest.mock("../components/landingNavbar");

const renderTree = (tree) => renderer.create(tree);
describe("<ForgotPassword>", () => {
  it("should render component", () => {
    expect(renderTree(<ForgotPassword />).toJSON()).toMatchSnapshot();
  });
});
