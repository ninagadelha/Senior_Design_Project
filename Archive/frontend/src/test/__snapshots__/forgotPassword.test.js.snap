// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForgotPassword> should render component 1`] = `
<div>
  <div
    className="main-content"
  >
    <div
      className="main-top"
      style={
        {
          "height": "100px",
          "marginBottom": "20px",
          "width": "100%",
        }
      }
    >
      <div
        className="header-about"
      >
        Forgot Password
      </div>
    </div>
    <div
      className="references-container"
    >
      <p>
        Email an administrator, and the administrator can reset your password.
      </p>
      <p>
        Make sure to contact the administrator with the email for which the password will be reset
      </p>
      <p>
        An administrator to contact is
         
        <a
          href="mailto:saba-ali@uiowa.edu"
        >
          saba-ali@uiowa.edu
        </a>
      </p>
    </div>
  </div>
</div>
`;
